CICOutputModels/OvRClassifier/ScaryCatScreeningML/v27/OvR_Result_5/OvR_Run_Report_v27.md

```
@@ -0,0 +1,24 @@
+## モデル詳細
+モデル名           : ScaryCatScreeningML
+ファイル生成日時   : 2025-06-01 10:51:08 +0900
+最大反復回数     : 20
+データ拡張       : なし
+特徴抽出器       : ScenePrint
+
+## トレーニング設定
+使用されたクラスラベル : sphynx, safe, human_hands_detected, black_and_white, mouth_open
+
+## 個別モデルの性能指標
+| クラス | 訓練正解率 | 検証正解率 | 再現率 | 適合率 | F1スコア |
```

markdownlint-cli2 (0.17.2)

Tables should be surrounded by blank lines
null

(MD058, blanks-around-tables)